CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ the code was deployed.
 
 ### Added
 
+- Language field to the add new client page of dashboard (CU-3bnggtt).
+- English and Spanish options to language field in new client page of dashboard (CU-3bnggtt).
```

Comment:
For my own understanding, do you mind clarifying the difference between these two points?